### DISCUSSION TITLE
Add '<topic>-insterest' labels for collaborators

### DISCUSSION THREAD

**Discussion by jerboaa** [oracle/graal#6595] [General] [answered]
Hi,

Red Hat would like to have a way to filter issues/PRs we are working on or are interested in. Over in the OpenJDK community there is a `redhat-interest` label that we can apply to bugs (and, thus, filter on). Do you think something like that would be possible for the GraalVM community too? Are there other possibilities for us to filter issues/prs based on our interest/work? If so, what would it be? Open to suggestions. Thanks!

/cc @zakkak @adinn

**Comment by alina-yur** [comment, marked as answer]
Hi @jerboaa. Sure, no problem. I created the "redhat-interest" label, you and several of your colleagues should have sufficient permissions to apply it to issues & PRs. lmk if it doesn't work.

**Reply by jerboaa** [reply]
@alina-yur That's fantastic. Thank you very much!

**Reply by alina-yur** [reply]
you are very welcome:)
